sentry-android-integration-tests/test-app-plain/build.gradle

```diff
@@ -0,0 +1,46 @@
+plugins {
```

Comment:
I'd keep using the `build.gradle.kts` style for better compatibility within the AS project.